doc/classes/WorkerThreadPool.xml

```diff
@@ -74,15 +74,15 @@
 		</method>
 		<method name="get_group_processed_element_count" qualifiers="const">
 			<return type="int" />
-			<param index="0" name="group_id" type="int" />
+			<param index="0" name="group_task_id" type="int" />
```

Comment:
You used find/replace and replaced too much

Reply:
I thought this should be renamed. 

"group task with the given ID" == "group_task_id", no?

Reply:
No that's outside the scope of this, please restore, this is just for documentaiton